webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2480,19 +2480,88 @@ var CURRENT_GROUP_DATA={};
                     _this.toggleMediaOptions(this);
 
                 });
-                $mck_text_box.keyup(function (){
+             
+            _this.cursorPosition =  function () {
+                var sel = document.getSelection();
+                sel.modify("extend", "backward", "documentboundary");
+                var pos = sel.toString().length;
+                if(sel.anchorNode != undefined) sel.collapseToEnd();
```

Comment:
@hardikparashar collapseToEnd doesn't work in IE.
https://developer.mozilla.org/en-US/docs/Web/API/Selection/collapseToEnd

Reply:
It works, just checked it on below link. Resolving the comment.
https://msdn.microsoft.com/en-us/ie/ff975174(v=vs.94)